autoload/flutter.vim

```diff
@@ -45,6 +45,20 @@ function! flutter#_exit_cb(job, status) abort
   call job_stop(a:job)
 endfunction
 
+function! flutter#_on_exit_nvim(job_id, data, event) abort dict
+  if exists('g:flutter_job')
+    unlet g:flutter_job
+  endif
+  call jobstop(a:job_id)
```

Comment:
I'm not sure if this is necessary in neovim

Reply:
Not 100% sure either - I don't use NV

Reply:
I think it is not necessary in `neovim` (on_exit is called when the job is already stopped):
https://github.com/neovim/neovim/issues/1598

But, I think that it would be similar with vim8? (`exit_cb` will be called, but `jobstop` may have been called already)
https://github.com/vim/vim/issues/817

As I gather, the distinction is between the command stopping (exit callback), and de-registering the job callbacks (jobstop).

Reply:
Yes, I actually think that called `job_stop` would be redundant, in hindsight. But AFAIK it's an idempotent operation, so this should be all right for now.